### ISSUE TITLE
ERROR: Index was outside the bounds of the array.

### ISSUE THREAD

**Issue by Dreamer269** [SubtitleEdit/subtitleedit#8174]
I have a folder with a bunch of vtt subtitles. I used the command line to batch convert all subtitles inside the folder.
Somehow, SubtitleEdit crashes at some point.

The command I used:
```
SubtitleEdit.exe /Convert *.vtt subrip /fixcommonerrors /MergeSameTimeCodes /MergeSameTexts /encoding:utf8 /inputfolder:"C:\subs"
```

Output:
```
Subtitle Edit 4.0.4 - Batch converter

1: Subtitle_157136af.vtt -> C:\subs\Subtitle_157136af.srt... done.
2: Subtitle_1b7f194c.vtt -> C:\subs\Subtitle_1b7f194c.srt... done.
3: Subtitle_1e236826.vtt -> C:\subs\Subtitle_1e236826.srt... done.
4: Subtitle_2101cd13.vtt -> C:\subs\Subtitle_2101cd13.srt... done.
5: Subtitle_22d92acf.vtt -> C:\subs\Subtitle_22d92acf.srt... done.

ERROR: Index was outside the bounds of the array.
   at Nikse.SubtitleEdit.Core.Forms.FixCommonErrors.FixMissingOpenBracket.RestoreMissingOpenParenthesis(String input)
   at Nikse.SubtitleEdit.Core.Forms.FixCommonErrors.FixMissingOpenBracket.Fix(Subtitle subtitle, IFixCallbacks callbacks)
   at Nikse.SubtitleEdit.Forms.FixCommonErrors.RunSelectedActions()
   at Nikse.SubtitleEdit.Forms.FixCommonErrors.RunBatch(Subtitle subtitle, SubtitleFormat format, Encoding encoding, String language)
   at Nikse.SubtitleEdit.Logic.CommandLineConvert.CommandLineConverter.RunActions(TextEncoding targetEncoding, Subtitle sub, SubtitleFormat format, List`1 actions, Boolean autoDetectLanguage)
   at Nikse.SubtitleEdit.Logic.CommandLineConvert.CommandLineConverter.BatchConvertSave(String targetFormat, TimeSpan offset, String deleteContains, TextEncoding targetEncoding, String outputFolder, String targetFileName, Int32 count, Int32& converted, Int32& errors, List`1 formats, String fileName, Subtitle sub, SubtitleFormat format, IList`1 binaryParagraphs, Boolean overwrite, Int32 pacCodePage, Nullable`1 targetFrameRate, ICollection`1 multipleReplaceImportFiles, List`1 actions, Nullable`1 resolution, Boolean autoDetectLanguage, BatchConvertProgress progressCallback, String ebuHeaderFile, String assaStyleFile, String ocrEngine, String preExt, Nullable`1 renumber, Nullable`1 adjustDurationMs, PreprocessingSettings preprocessingSettings, CancellationToken cancellationToken)
   at Nikse.SubtitleEdit.Logic.CommandLineConvert.CommandLineConverter.Convert(String[] arguments)


5 file(s) converted in 00:00:04.3617883
```

Here are the sub files do to some testing/debugging:
[subs.zip](https://github.com/SubtitleEdit/subtitleedit/files/14958820/subs.zip)

**Comment by niksedk** [comment]
Thx - should be fixed in 4.0.5 :)

**Comment by niksedk** [comment]
@ho3innn: Could you attach or email the file that causes this error?
